diff --git a/src/server/server.pro b/src/server/server.pro
index d5005c2..efa7745 100644
--- a/src/server/server.pro
+++ b/src/server/server.pro
@@ -10,7 +10,8 @@ INCLUDEPATH += ../h
 
 QMAKE_LIBDIR_FLAGS += -L../lib -L../voland
 LIBS += -ltimed -ltimed-voland
-CONFIG += qmlog
+CONFIG += link_pkgconfig qmlog
+PKGCONFIG += libcreds3
 
 IODATA_TYPES = queue.type config.type settings.type customization.type tzdata.type
 
