diff --git a/src/server/timed.cpp b/src/server/timed.cpp
index 601a394..def3319 100644
--- a/src/server/timed.cpp
+++ b/src/server/timed.cpp
@@ -170,7 +170,7 @@ void Timed::init_scratchbox_mode()
   scratchbox_mode = path && strstr(path, "scratchbox") ; // XXX: more robust sb detection?
   log_info("%s" "SCRATCHBOX detected", scratchbox_mode ? "" : "no ") ;
 #else
-  scratcbox_mode = false ;
+  scratchbox_mode = false ;
 #endif
 }
 
